<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentBaptismActivity</class>
 <widget class="QWidget" name="AgentBaptismActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,0,0,0,0,10,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelBodyWelcome">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;Delivery&lt;/span&gt;&lt;/p&gt;&lt;p&gt;You are about to bring this OctoMYâ„¢ &lt;span style=&quot; font-weight:600;&quot;&gt;Agent&lt;/span&gt; into the world.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRandomName">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../libres/resources/icons.qrc">
         <normaloff>:/icons/roll_dice.svg</normaloff>:/icons/roll_dice.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QComboBox" name="comboBoxGender">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Leave to chance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Genderless</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRandomGender">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../libres/resources/icons.qrc">
         <normaloff>:/icons/roll_dice.svg</normaloff>:/icons/roll_dice.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonDone">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../libres/resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>AgentBaptismActivity</receiver>
   <slot>done()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRandomName</sender>
   <signal>clicked()</signal>
   <receiver>AgentBaptismActivity</receiver>
   <slot>randomizeName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRandomGender</sender>
   <signal>clicked()</signal>
   <receiver>AgentBaptismActivity</receiver>
   <slot>randomizeGender()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>randomizeName()</slot>
  <slot>randomizeGender()</slot>
  <slot>done()</slot>
 </slots>
</ui>
